src/writer/sync.py

```
@@ -248,36 +248,48 @@
                 # Process file deletions
                 if self.operation_queues.files_to_delete:
                     self.log(f"Deleting {len(self.operation_queues.files_to_delete)} files")
                     for file_path in self.operation_queues.files_to_delete.copy():
-                        to = self.dest_path(file_path)
-                        self.remove_path(to)
-                        self.log("File removed:", self.rel(file_path))
+                        try:
+                            self.log("File remove:", self.rel(file_path))
+                            to = self.dest_path(file_path)
+                            self.remove_path(to)
+                        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)